app/models/service_provider.rb

```diff
@@ -64,6 +64,7 @@ def self.possible_attributes
       phone
       x509_subject
       x509_presented
+      verified_at
```

Comment:
can we add a spec to make sure this gets used where we think it does? like a view spec or a controller spec?